plugin.video.BlanketFort/addon.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.BlanketFort"
+       name="BlanketFort"
+       version="1.0.0"
+       provider-name="BF Media">
+  <requires>
+    <import addon="xbmc.python" version="2.1.0"/>

```

Comment:
version needs to be 2.24.0 for jarvis. Pls see http://kodi.wiki/view/addon.xml#Dependency_versions